### PR TITLE
Feature/create booking package 58

### PR DESCRIPTION
**Pull request by artiko00**
Hi! we were added jest eslint and prettier! best regards! have a nice weekend!

### CODE REVIEW

**Comment by artiko00** [comment]
#58

**Review by elyager — CHANGES_REQUESTED** [review]
There more changes required, see this [example](https://github.com/Platzi-Master-C9/booking-services/pull/117/files)

**Comment by artiko00** [comment]
@elyager Hi! can you check the PR again? thanks!

File: packages/admin-panel/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+coverage
```
Comment (JoseNoriegaa) [inline packages/admin-panel/.gitignore:1]: Can you add EOL here?

File: packages/api-gateway/src/drivers/http/adapters/admin_panel/handlers.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	const result = await this.adminPanelService.sayHello();
+
+	return reply.code(200)
+		.header('Content-Type', 'application/json; chartset:utf-8')
+		.send({ result });
+}
+
+module.exports = {
+	sayHello
+}
```
Comment (JoseNoriegaa) [inline packages/api-gateway/src/drivers/http/adapters/admin_panel/handlers.js:13]: Can you add EOF here?

File: packages/api-gateway/src/drivers/http/adapters/booking/handlers.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+	req.log.info('[http-server]: Booking Module');
+
+	return reply.code(200)
+		.header('Content-Type', 'application/json; chartset:utf-8')
+		.send({ message: 'Hello Booking Module!' });
+}
+
+module.exports = {
+	initBookings
+}
```
Comment (JoseNoriegaa) [inline packages/api-gateway/src/drivers/http/adapters/booking/handlers.js:14]: EOF

File: packages/api-gateway/src/drivers/http/routes/admin_panel/api/index.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+const { adminPanelAdapters } = require('../../../adapters');
+//const schema = require('./schema');
+
+async function adminPanelRouter(fastify) {
+
+	await fastify.get('/',adminPanelAdapters.sayHello);
+}
+
+module.exports = adminPanelRouter;
```
Comment (JoseNoriegaa) [inline packages/api-gateway/src/drivers/http/routes/admin_panel/api/index.js:11]: EOF

File: packages/booking/src/index.js [hunk 5]
```diff
@@ -0,0 +1 @@
+console.log("Booking squad")
```
Comment (JoseNoriegaa) [inline packages/booking/src/index.js:1]: EOF

File: packages/booking/package.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/artiko00/booking-services.git"
+  },
+  "scripts": {
+    "test": "echo \"Error: run tests from root\" && exit 1"
```
Comment (JoseNoriegaa) [inline packages/booking/package.json:24]: I noticed that you added the jest.config file, but you don't added the "test" command to run the tests.

**Review by JoseNoriegaa — CHANGES_REQUESTED** [review]
I noticed that this PR is outdated, so please, rebase this PR, and make sure you run `yarn lint:fix` inside the api-gateway package and fix the eslint issues that may appear.

**Comment by gojideth** [comment]
Hi! I changed the requested changes to be able to merge this PR. #58

File: packages/api-gateway/src/drivers/http/adapters/index.js [hunk 7]
```diff
@@ -1,9 +1,17 @@
...
 module.exports = {
-	mathAdapters,
-	statusAdapters,
-}
+  mathAdapters,
+  statusAdapters,
+  adminPanelAdapters,
+  notificationsAdapters,
+  placesAdapters,
+  bookingAdapters,
```
Comment (JoseNoriegaa) [inline packages/api-gateway/src/drivers/http/adapters/index.js:16]: Looks like you removed accidentally the geolocationAdapters from the exported modules, Can you added again?
```suggestion
  bookingAdapters,
  geolocationAdapters,
```

File: packages/api-gateway/src/drivers/http/adapters/index.js [hunk 8]
```diff
@@ -1,9 +1,17 @@
-'use strict';
-
+// Internal dependencies
 const mathAdapters = require('./math/handlers');
+const geolocationAdapters = require('./geolocation/handlers');
 const statusAdapters = require('./status/handlers');
+const adminPanelAdapters = require('./admin_panel/handlers');
+const notificationsAdapters = require('./notifications/handlers');
+const placesAdapters = require('./places/handlers');
+const bookingAdapters = require('./booking/handlers');
```
Comment (JoseNoriegaa) [inline packages/api-gateway/src/drivers/http/adapters/index.js:8]: Missing reference to messageAdapters
```suggestion
const bookingAdapters = require('./booking/handlers');
const messageAdapters = require('./messages/handlers');
```

**Review by JoseNoriegaa — CHANGES_REQUESTED** [review]
There are a couple of details in the api-gateway, probably due to merge commits, I recommend you use rebase instead of merge so you don't ended up with a lot of commits in you PR because right now your PR is a bit difficult to review because we don't know what you actually modified/added or what just added in merge commits.

File: packages/booking/package.json [hunk 9]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@booking-services/booking",
+  "version": "1.0.0",
+  "description": "This package contains all bussiness logic from booking squad",
+  "author": "Booking Squad",
+  "homepage": "https://github.com/Platzi-Master-C9/backend-services/tree/main/packages/booking#readme",
+  "license": "ISC",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
```
Comment (JoseNoriegaa) [inline packages/booking/package.json:10]: Can you change this to `src`?
```suggestion
    "src": "src",
```

File: packages/booking/package.json [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
```
Comment (JoseNoriegaa) [inline packages/booking/package.json:14]: Can you change this to `src`?
```suggestion
    "src"
```

File: packages/booking/package.json [hunk 11]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@booking-services/booking",
+  "version": "1.0.0",
+  "description": "This package contains all bussiness logic from booking squad",
+  "author": "Booking Squad",
+  "homepage": "https://github.com/Platzi-Master-C9/backend-services/tree/main/packages/booking#readme",
+  "license": "ISC",
+  "main": "index.js",
```
Comment (JoseNoriegaa) [inline packages/booking/package.json:8]: Can you change this to `src/index.js`?
```suggestion
  "main": "src/index.js",
```

**Comment by gojideth** [comment]
I changed to src instead of lib :)

**Review by JoseNoriegaa — APPROVED** [review]
LGTM

**Review by elyager — APPROVED** [review]

**Pull request merged**